ts/components/MemberListItem.tsx

```diff
@@ -114,7 +118,7 @@ export const MemberListItem = (props: {
     >
       <StyledInfo>
         <AvatarItem memberPubkey={pubkey} isAdmin={isAdmin || false} />
-        <StyledName>{memberName}</StyledName>
+        <StyledName maxNameWidth={maxNameWidth}>{memberName}</StyledName>
```

Comment:
should probably be renamed to maxWidth once we are targetting the StyledName element.
I.e. keep   `maxNameWidth?: string;` on line 79, but change line 121 to 
`        <StyledName maxWidth={maxWidth}>{memberName}</StyledName>`

Reply:
Fixed